### DISCUSSION TITLE
i18n lookup

### DISCUSSION THREAD

**Discussion by alec-c4** [Shopify/packwerk#413] [Q&A]
Hey!
I'd like to add packwerk to my project, but there is an issue with i18n keys lookup. Earlier I used key `admin.users.edit.title`, but now I need to use `admin.views.admin.users.edit.title` for package views, located in `app/packages/admin/views/edit.html.erb`. Is there any ability to use `t('.title')` in the package views and to lookup `admin.users.edit.title` in the `app/packages/admin/config/locales/*.yml`??

**Comment by tylerwillingham** [comment]
@alec-c4 this feels like it's probably not related to packwerk since shopify/packwerk is for static analysis. How are you loading your files under app/packages? Are you using other dependencies?

One thing that stood out to me about your folder structure as unexpected is that your views subdir isn't under app, it's a sibling to where app/ would be. If you're using a mechanism for loading from app/packages sub directories that mirrors rails' zeitwerk loader, that could be a problem
